multiraft/heartbeat_test.go

```diff
@@ -331,7 +328,6 @@ func TestHeartbeatMultipleGroupsJointLeader(t *testing.T) {
 			expCntSecondPhase, actCnt)
 	}
 	// Keep processing without inspection and shutdown cluster.
-	stopper.AddWorker()
 	go processEventsUntil(transport.Events, stopper, alwaysFalse)
```

Comment:
Sure this is correct? Why not add a worker here? Maybe use `stopper.RunWorker()`, as you've done elsewhere in this file.